### PR TITLE
Sprint3 code coverage

### PR DESCRIPTION
**Pull request by EdRW**
Added support for 2 separate code coverage tools. Both are integrated into Travis CI. I'm sure which one I like more yet. I figure there's no harm in having more than one for now.
- [Coveralls](https://coveralls.io/github/PhotoTagger/django-initial)
- [Codecov](https://codecov.io/gh/PhotoTagger/django-initial)

### CODE REVIEW

**Review by arunach2 — DISMISSED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -12,20 +12,23 @@ install:
...
+- pip install codecov
+
 # build notifications
 notifications:
     slack: se491-fall2018:ccqfPhe1cEHeZJlQgCOe3m8C
     
 script:
 #  run tests
-  - python manage.py test
+  - coverage run --source='.' manage.py test
```
Comment (kmcdon10) [inline .travis.yml:30]: Can you add this to the readme?

**Comment by kmcdon10** [comment]
Do the code coverage libraries support excluding certain files or directories? We should be excluding files we didn't write, such as tfmodels, tfutils, and the tests themselves

**Comment by kmcdon10** [comment]
The UI for codecov is pretty cool. I'm fine with having both for now so long as we can add some kind of pre-commit hook that prevents us from committing code that decreases the code coverage

File: .travis.yml (hunk 1 above)
Comment (EdRW) [reply .travis.yml:30]: No problemo

**Comment by EdRW** [comment]
Coverage allows us to exclude specified files. It is also possible to exclude specified code branches like if/else statements.

Some of  the reasons I had a hard time choosing between coveralls and codecov is that coveralls makes it easy to add requirements for code coverage percentage on pull requests. But codecov has better tracking of branch coverage. So I figure we can play with both until we determine which is obviously better

**Review by kmcdon10 — DISMISSED** [review]

**Comment by EdRW** [comment]
Sorry, need aprovals again. Did a rebase.

**Review by kmcdon10 — APPROVED** [review]

**Review by JAM715 — APPROVED** [review]

**Pull request merged**